src/services/subsocial/evmAddresses/mutation.tsx

```diff
@@ -62,7 +65,9 @@ export function useLinkEvmAddress({
         getContext: () => '',
         onStart: () => setOnCallbackLoading(true),
         onSuccess: async ({ address }) => {
-          await getLinkedEvmAddressQuery.fetchQuery(client, address)
+          await mutateAccountsDataCache(address)
+          await getAccountDataQuery.fetchQuery(client, address)
```

Comment:
maybe its better to use invalidate?

Reply:
the invalidate function is not async. So in my case it's better to use fetchQuery because it has to wait for the function to complete